quisp/modules/QRSA/RuleEngine/RuleEngine.h

```diff
@@ -67,12 +67,15 @@ class RuleEngine : public IRuleEngine {
   IRealTimeController *realtime_controller;
   int *qnic_burst_trial_counter;
   qnicResources *allResources;  // Size will be defined in initialization. If 3 qnic types, then size is 3. Type defined in QUBIT.h
+  std::map<int, photonTransmissionConfig> transmission_interface;
```

Comment:
add a comment for `int`. it's too hard to guess what the int value means.

Reply:
what do you mean "interface"? is that an interface for something? 
this name seems misleading.